### DISCUSSION TITLE
dev branch and milestones, how to separate updates?

### DISCUSSION THREAD

**Discussion by marcobarilari** [cpp-lln-lab/bidspm#232] [Ideas]
(let's try this discussion feature)

would having a dev branch per milestone be more optimal for release jumps?

For example, if I feel like working for an update that fit in the release 1.0.0 but the code is in preparation for 0.2.0, should be preferable that (1) my pr related to 1.0.0 stay sleepy until the 0.2.0 jump or (2) it is better to have a dev_0.2.0 and dev_1.0.0?

**Comment by Remi-Gau** [comment]
Ideally yes I would like to do it that way indeed.

https://blog.axosoft.com/gitflow/
